style/hudl_sql_style_guide.md

```diff
@@ -45,9 +50,24 @@ description: A guide to writing clean, clear, and consistent SQL.
   __BAD__:
   `SELECT ksr.backings AS b`
 
-* Use [Common Table Expressions](http://www.postgresql.org/docs/8.4/static/queries-with.html) (CTEs) early and often, and name them well.
-* All dates should be formatted as 'YYYY-MM-DD'
-* Use `!=` for the inequality operator
+## `Common Table Expressions (CTEs)`
+
+* Use [Common Table Expressions](http://www.postgresql.org/docs/8.4/static/queries-with.html) often, and place them at the top
+* CTEs should be used to represent source tables to make code more readable
```

Comment:
is `source` the right word here?  or would `refs` make more sense?  To me, I think of the source tables only.

Reply:
Per your suggestion added wording to make clearer.